apps/omg_watcher/test/omg_watcher/exit_processor/core/state_interaction_test.exs

```diff
@@ -289,6 +289,23 @@ defmodule OMG.Watcher.ExitProcessor.Core.StateInteractionTest do
     assert [_] = Core.get_active_in_flight_exits(processor)
   end
 
+  test "deleting in-flight exits works with State",
+       %{processor_empty: processor, state_empty: state, alice: alice} do
+    ife_exit_tx = TestHelper.create_recovered([{1, 0, 0, alice}], @eth, [{alice, 9}])
+    ife_id = 1
+
+    state = TestHelper.do_deposit(state, alice, %{amount: 10, currency: @eth, blknum: 1})
+    {:ok, _, state} = State.Core.form_block(state)
+
+    {_processor, deleted_utxos, _db_updates} =
+      processor
+      |> start_ife_from(ife_exit_tx, exit_id: ife_id)
+      |> Core.delete_in_flight_exits([%{exit_id: ife_id}])
+
+    assert {:ok, {[{:delete, :utxo, _}], {[{:utxo_position, 1, 0, 0}], []}}, _} =
```

Comment:
slightly curious, what will be in the `_` part and why we don't match it to detail?

Reply:
@boolafish  Key for utxo to delete from the db. We don't match it, because it's not a unit test that tests `State.exit_utxos` (it's tested in detail in other file). We just wan to know if what `delete_in_flight_exit` returns fits into `State.exit_utxos` and do just basic sanity check, hence matching the returned utxo.